packages/migration/migrations/20220817082035-informant-legal-guardin-doc.js

```diff
@@ -0,0 +1,40 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * OpenCRVS is also distributed under the terms of the Civil Registration
+ * & Healthcare Disclaimer located at http://opencrvs.org/license.
+ *
+ * Copyright (C) The OpenCRVS Authors. OpenCRVS and the OpenCRVS
+ * graphic logo are (registered/a) trademark(s) of Plan International.
+ */
+export const up = async (db, client) => {
+  const session = client.startSession()
+    try {
```

Comment:
Is prettier in use in this package? The indentation here before `try` looks odd

Reply:
right the file matcher actually didn't include .js files. I'll have to add it in. Thanks